src/zoid/card-fields/component.jsx

```diff
@@ -245,6 +247,14 @@ export const getCardFieldsComponent: () => CardFieldsComponent = memoize(
             value: ({ props }) => props.parent.props.createOrder,
           },
 
+          ...(isPayPalDomain && {
```

Comment:
Why do we require this check `isPayPalDomain` ?

Reply:
Because according to @imbrian Brian Tedder, we need to ensure this props is only usable in stage, and not usable in production or sandbox.

Reply:
We do need to actually invoke `isPayPalDomain` though, as it is a function.

Reply:
We also have [protectedExports](https://github.com/paypal/paypal-checkout-components/blob/main/src/lib/security.js#L28) if you're just validating Paypal domain

Reply:
@9zees / @imbrian - I don't think that `isPayPalDomain()` will work here for the sake of preventing this callback being available in sandbox/production. The cardFields component would be rendered inside the zoid iFrame and the domain will still be `*.paypal.com`. Maybe an explicit check for the environment would be better here.

Reply:
This is sorta true. This code runs first on the merchant domain to create the iframe, so this would prevent passing the callback to the iframe. Ultimately, this is client-side code, so this just offers some guardrails against accidental misuse.